pkg/epinio/pages/c/_cluster/applications/createapp/index.vue

```diff
@@ -89,6 +89,15 @@ export default Vue.extend<Data, any, any, any>({
     };
   },
 
+  watch: {
+    source: {
+      handler() {
+        this.steps[0].ready = !!this.source?.type;
```

Comment:
ready step depends on validation. if you put a console.log here it fires a lot (for instance when a user has selected github and entered anything for the username the step becomes enabled)